packages/server/src/backfill/separate-first-last-names.command.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { Command } from 'nestjs-command';
+import { UserModel } from 'profile/user.entity';
+
+@Injectable()
+export class BackfillSeparateFirstLastNames {
+  @Command({
+    command: 'backfill:first-last-names',
+    describe: 'change all names to first and last names',
+    autoExit: true,
+  })
+  async fix(): Promise<void> {
+    const users = await UserModel.find();
+    users.forEach((user) => {
+      try {
+        user.firstName = user.name.split(' ')[0];
+        user.lastName = user.name.split(' ').slice(1).join(' ');
+      } catch (e) {
+        user.firstName = user.name;
+      }
```

Comment:
What if we created a list of names that error which we can log after the script completes? I'm hoping their will at most be only a few that error, and if we have the list we can look them up fix it manually.

Reply:
good idea -- adding it